diff -Naur xmlsectool.orig/xmltool.sh xmlsectool/xmltool.sh
--- xmlsectool.orig/xmltool.sh	2011-02-18 16:50:21.000000000 +0100
+++ xmlsectool/xmltool.sh	2011-02-18 16:49:58.000000000 +0100
@@ -6,8 +6,7 @@
 declare LOCALCLASSPATH
 declare LIBDIR
 
-LOCATION=$0
-LOCATION=${LOCATION%/*}
+LOCATION=/usr/share/java/xmlsectool
 
 if [ -z "$JAVA_HOME" ] ; then
   echo "ERROR: JAVA_HOME environment variable is not set."
@@ -30,7 +29,7 @@
 LOCALCLASSPATH=$JAVA_HOME/lib/tools.jar:$JAVA_HOME/lib/classes.zip
 
 # add in the dependency .jar files from the lib directory
-LIBDIR=$LOCATION/lib
+LIBDIR=$LOCATION
 LIBS=$LIBDIR/*.jar
 for i in $LIBS
 do
